test/ranges.jl

```diff
@@ -838,3 +838,10 @@ let r = linspace(-big(1.0),big(1.0),4)
     @test isa(@inferred(r[2]), BigFloat)
     @test r[2] ≈ big(-1.0)/3
 end
+
+let r = linspace(1.0, 3+im, 4)
```

Comment:
May I suggest https://github.com/jayphelps/git-blame-someone-else, disable force push protection and literally rewrite history!